9control.tex

```diff
@@ -677,28 +677,28 @@
 	 &  & 0 = not used; \\
 	 &  & 1 = invoke female to male age-specific function; and \\
 	 &  & -1 = invoke male to female age-specific function. \\
-	 &  & Note: this creates the annual, age-specific fraction that change sex, it is not the fraction that is each sex. \Bstrut\\
+	 &  & Note: this creates the annual, age-specific fraction that change sex, it is not the fraction that is each sex. Also, the fraction female parameter should be configured to reflect the fraction of age 0 fish. Values of 0 or 1 may lead to NaN likelihoods, so we recommend changing values that would be fixed at 0 or 1 to 0.000001 or 0.999999.\Bstrut\\
```

Comment:
@e-perl-NOAA You're right. I wanted that information as close to the top of the hermaphroditism section as possible, but it would just get lost in that note. Instead, I've created a new intro section in 21ef32a27a3f32954bcfbd0b4bb0f522e670e102 prior to the table of control file settings which hopefully provides better overall information on the options and moved all the notes into that paragraph.